domain/iam.go

```diff
@@ -11,6 +11,8 @@ type IAMConfig struct {
 	Provider IAMProviderType   `json:"provider" yaml:"provider" validate:"required,oneof=http shield"`
 	Config   interface{}       `json:"config" yaml:"config" validate:"required"`
 	Schema   map[string]string `json:"schema" yaml:"schema"`
+	// AllowRequestError is a flag that lets the appeal creation continue even if the request to the IAM provider fails.
+	AllowRequestError bool `json:"allow_request_error" yaml:"allow_request_error"`
```

Comment:
`allow_request_error` sounds too generic for this use cases. In case we will have another use case that use IAM, this will make it confusing. How about we think about the other naming that is specific for appeal creation?
`allow_appeal_creation_without_iam` ?

Reply:
I think putting this config in policy's appeal config is more appropriate, 
https://github.com/goto/guardian/blob/2d90d1b91f18145799a53f8437da47e8f2fb555c/domain/policy.go#L285 with field name `ignore_iam_error`
@mabdh @bsushmith wdyt?

Reply:
yup lets do it

Reply:
renaming this field to `allow_creator_details_failure` and add more description for the field. Please check @mabdh @bsushmith